app/assets/stylesheets/custom.css.scss

```diff
@@ -1,7 +1,28 @@
 .font-size-22 {
   font-size: 22px;
 }
-
+.notice {
+  display: block;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, display, font-size, line-height, margin, padding